exir/graph_module.py

```diff
@@ -68,3 +68,25 @@ def get_control_flow_submodules(
             control_flow_submodules.append(_get_submodule(graph_module, node, 0))
 
     return control_flow_submodules
+
+# TODO(gasoonjia): remove this and leverage core pytorch bfs_trace_with_node_process after code freeze
+def bfs_trace_with_node_process(
+    gm: torch.fx.GraphModule, node_op: Callable[[torch.fx.Node], None]
```

Comment:
this one takes `gm`? I remember another one takes `ep`? maybe align?

Reply:
There're two `bfs_trace_with_node_process`, one is under ao, the other is under et.
ao's take ep and gm while et's only takes gm.